### DISCUSSION TITLE
Add an option to force disable the Great (aka. 300) text when pressing a circle.

### DISCUSSION THREAD

**Discussion by NukeThief** [ppy/osu#21001] [Ideas] [answered]
This is could actually be very useful for some skins that still show the Great/300 text.

**Comment by mk56-spn** [comment, marked as answer]
https://youtu.be/zOJKWJA-vog?t=85

i have the feeling this has been answered a bunch of times before but i cant find the relevant discussions, already planned as shown by the timestamp provided of the latest lazer update video

**Reply by NukeThief** [reply]
yipeee :D
